perf/runner.js

```diff
@@ -215,23 +219,35 @@ async function runInBrowser(browser, page, options) {
     return;
   }
 
-  logLine(
-    `Running ${benchmarks.length} benchmarks on ${browserName(browser)}...`,
-  );
+  const jsonEntries = [];
+  if (options.format !== 'json') {
+    logLine(
+      `Running ${benchmarks.length} benchmarks on ${browserName(browser)}...`,
+    );
+  }
   for (const benchmark of benchmarks) {
-    const testResult = await page.evaluate(
+    const result = await page.evaluate(
       ({name, group, format}) =>
         // eslint-disable-next-line no-undef
         runBenchmarkByNameAndGroup(name, group, format),
       {format: options.format, ...benchmark},
     );
-    if (testResult === undefined) {
-      continue;
+    if (result) {
+      if (result.error) {
+        process.stderr.write(result.error + '\n');
+      } else {
+        jsonEntries.push(result.jsonEntry);
+        if (options.format !== 'json') {
```

Comment:
I'm wondering if we should just make `logLine` a noop if format is json?

Reply:
done